task3.py

```diff
@@ -0,0 +1,45 @@
+'''
+3. You are given a list of sentences. Create a word tree from each sentence and find how many times the word is appeared in other word trees.
+ Input:
+sentences = `[“My name is Ram”, “”He is a good person”, “You should be careful while coding”, “He can do better”, “The person is mysterious”, “Jay Shree Ram”, “It is my pen.”]`
+ Output:
+word_trees = [[“My”, “name”, “is”, “Ram”], [“He”, “is”, “a”, “good”, “person”], [“You”, “should”, “be”, “careful”, “while”, “coding”], [“He”, “can”, “do”, “better”], [“The”, “person”, “is”, “mysterious”], [“Jay”, “Shree”, “Ram”], [“It”, “is”, “my”, “pen”]]
+Number of time each word appears:
+{“My”: 1, “name”: 1, “is”: 4, “Ram”: 2, “He”: 2, “good”: 1, “person”: 2, “You”: 1 ......} Likewise all word should be covered.
+'''
+# List
+sentences = ["My name is Ram","He is a good person","You should be careful while coding","He can do better","The person is mysterious","Jay Shree Ram","It is my pen"]
+
+#list of list 
+list_of_list= []
+for element in sentences:
+    sub = element.split(', ')
+    list_of_list.append(sub)    
+print(list_of_list)
+
+#list of list of string
+list_of_string=[]
+for list in list_of_list:
+    for string in list:
+        convert= string.split(' ')
+        list_of_string.append(convert)
+print(list_of_string)
+
+#list of list of string convcert to one list
+flat_list = []
+for sublist in list_of_string:
+    for item in sublist:
+        flat_list.append(item)
+
+#Count the word
+from collections import Counter
+words_to_count = (word for word in flat_list)
+count= Counter(words_to_count)
+cout_word=dict(count.most_common(100000))
```

Comment:
Please explain me why you have used 100000?

Reply:
count the max 100000 words